scripts/release-with-submodule.ts

```diff
@@ -0,0 +1,42 @@
+import { execSync } from 'child_process';
+import { readFileSync } from 'fs';
+import { join } from 'path';
+
+const submoduleDir = 'enterprise/packages';
+
+function runCommand(command: string, directory?: string) {
+  console.log(`Running command: ${command}${directory ? `directory:${directory}` : '.'}`);
+  execSync(command, {
+    stdio: 'inherit',
+    cwd: directory,
+  });
+}
+
+try {
+  console.log('Step 1: Run lerna version in monorepo without committing or creating a release');
+  runCommand('pnpm lerna version patch --yes --no-git-tag-version --no-push --force-publish');
```

Comment:
```suggestion
  runCommand('pnpm lerna version patch --yes --no-git-tag-version --no-push ');
```

needed to be removed, i added it only for local testing.